pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -54,8 +55,15 @@ const (
 	// The minimum memory limit allocated to docker container: 150Mi
 	MinDockerMemoryLimit = 150 * 1024 * 1024
 
-	dockerProcessName = "docker"
-	dockerPidFile     = "/var/run/docker.pid"
+	dockerProcessName     = "docker"
+	dockerPidFile         = "/var/run/docker.pid"
+	containerdProcessName = "docker-containerd"
+	containerdPidFile     = "/run/docker/libcontainerd/docker-containerd.pid"
```

Comment:
Is using this pid file documented and stable? We should probably ask the Docker team to do that if not.